examples/device/net_lwip_webserver/src/usb_descriptors.c

```diff
@@ -155,15 +155,16 @@ static uint16_t _desc_str[32];
 // Application return pointer to descriptor, whose contents must exist long enough for transfer to complete
 uint16_t const* tud_descriptor_string_cb(uint8_t index, uint16_t langid)
 {
-  (void)langid;
+  (void) langid;
 
-  unsigned chr_count = 0;
+  uint8_t chr_count = 0;
 
   if (STR_LANGID == index)
   {
     memcpy(&_desc_str[1], string_desc_arr[0], 2);
     chr_count = 1;
   }
+#if CFG_TUD_NET == OPT_NET_ECM
```

Comment:
This code should not be #if conditional.  It only seems to be prudent because I failed to do this in desc_device:

```
    .iManufacturer      = STR_MANUFACTURER,
    .iProduct           = STR_PRODUCT,
    .iSerialNumber      = STR_MAC,
```